projects/commudle-admin/src/app/feature-modules/community-builds/components/community-build-details/community-build-details.component.html

```diff
@@ -51,6 +51,13 @@ <h1>
                                      [socialMediaLinks]="true"></app-user-profile-horizontal>
       </div>
 
+      <div class="teammate-details" *ngIf="contributors.length > 0">
+        <p>teammates</p>
+          <div *ngFor="let contri of contributors;">
```

Comment:
contri??
contributor is not a word we've used anywhere, let's make it teammates and teammate